tests/integration/pilot/multi_version_revision_test.go

```diff
@@ -165,6 +169,33 @@ func installRevisionOrFail(t framework.TestContext, version string, configs map[
 	}
 }
 
+// ReadInstallFile reads a tar compress installation file from the embedded
+func ReadInstallFile(f string) (string, error) {
```

Comment:
I think this is a merge conflict

Reply:
great catch. I do not know how you caught this in these 200 file changes